src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -166,23 +166,23 @@ def exec(self, shell_command: str, container: str = None) -> str:
 
         return exec_shell_command(self.metadata.name, shell_command, self.metadata.namespace, container)
 
-    def get_logs(
-        self,
-        container=None,
-        previous=None,
-        tail_lines=None,
-        regex_replacer_patterns: Optional[List[NamedRegexPattern]] = None,
-        regex_replacement_style: Optional[RegexReplacementStyle] = None,
-        filter_regex: Optional[str] = None,
+    @staticmethod
+    def find_logs(
+            name: str,
+            namespace: str,
+            container=None,
+            previous=None,
+            tail_lines=None,
+            regex_replacer_patterns: Optional[List[NamedRegexPattern]] = None,
+            regex_replacement_style: Optional[RegexReplacementStyle] = None,
+            filter_regex: Optional[str] = None,
     ) -> str:
         """
         Fetch pod logs, can replace sensitive data in the logs using a regex
         """
-        if container is None:
```

Comment:
I would re-add this and remove 
`container = self.spec.containers[0].name
` 
from the other to places in this PR